In Solaris 12, we will be normalizing man pages. See PSARC 2015/428.
This means that what was section 1m, is now section 8 and what was
section 4 is now section 5. But currently only for Solaris 12.

This patch should eventually be sent upstream, but for that to happen
it will need to be reworked into a form that will be acceptable for
all versions of Solaris. Until then, we are just using this Solaris 12
specific patch.

--- sudo-1.8.9p5/configure.ac.orig	2015-10-20 07:25:44.363830767 -0700
+++ sudo-1.8.9p5/configure.ac	2015-10-20 07:26:11.354071075 -0700
@@ -1640,8 +1640,8 @@
 		if test "$with_AFS" = "yes"; then
 		    AFS_LIBS="-lc -lucb"
 		fi
-		: ${mansectsu='1m'}
-		: ${mansectform='4'}
+		: ${mansectsu='8'}
+		: ${mansectform='5'}
 		test -z "$with_pam" && AUTH_EXCL_DEF="PAM"
 		AC_CHECK_FUNCS(priv_set, [PSMAN=1])
 		;;
